packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/CsvAsset.java

```diff
@@ -0,0 +1,33 @@
+package software.uncharted.terarium.hmiserver.models.dataservice;
+
+import lombok.Data;
+import lombok.experimental.Accessors;
+import lombok.extern.slf4j.Slf4j;
+import software.uncharted.terarium.hmiserver.annotations.TSModel;
+import software.uncharted.terarium.hmiserver.annotations.TSOptional;
+
+import java.io.Serializable;
+import java.util.*;
+
+@Data
+@Accessors(chain = true)
+@Slf4j
+@TSModel
+public class CsvAsset implements Serializable {
+	List<List<String>> csv;
+	@TSOptional
+	List<CsvColumnStats> stats;
+	List<String> headers;
+
+	public CsvAsset(List<List<String>> csv, List<CsvColumnStats> stats, List<String> headers){
```

Comment:
Having explicit constructors is fine, but with using Lombok we can technically avoid these.  If you add the annotation `@AllArgsConstructor` Lombok will auto create a constructor for all the fields on this class (`csv`, `stats`, `headers`)

Reply:
Annotation magic scares cavemen like me